<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>912</width>
    <height>591</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TeRI</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="resultWindow">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>601</width>
      <height>391</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="button_close">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>520</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Zamknij</string>
    </property>
   </widget>
   <widget class="QPushButton" name="browse_txt_button">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>420</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QPushButton" name="browse_xml_button">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>470</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="path_txt_window">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>471</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="path_xml_window">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>470</y>
      <width>471</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>520</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="logsButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>520</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Logi</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="excelSheets">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>10</y>
      <width>241</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>Wybierz zakładkę z excel`a</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>171</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>2G</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3G</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>AT&amp;T</string>
      </property>
     </item>
    </widget>
    <widget class="QTextBrowser" name="textBrowser">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>50</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser_2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>100</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser_3">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>150</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>912</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionZamknij">
   <property name="text">
    <string>Zamknij</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>button_close</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>556</x>
     <y>559</y>
    </hint>
    <hint type="destinationlabel">
     <x>620</x>
     <y>574</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
